.docker/app/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+rm -f /app/.internal_test_app/tmp/pids/server.pid
+bundle install
+exec bundle exec rails s -p 3000 -b 0.0.0.0
```

Comment:
I believe if we tie in rake jhu:server here, we should get the project test fixtures indexed as part of the docker spin up.

Reply:
I can take a closer look at it after this metadata workflow meeting at 1pm CT / 2pm ET.